common/src/main/java/gov/cms/ab2d/common/repository/AttestationRepository.java

```diff
@@ -1,10 +1,12 @@
 package gov.cms.ab2d.common.repository;
 
 import gov.cms.ab2d.common.model.Attestation;
+import gov.cms.ab2d.common.model.Contract;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface AttestationRepository extends JpaRepository<Attestation, Long> {
 
+    Attestation findOneByContractOrderByAttestedOnDesc(Contract contract);
```

Comment:
Since we don't need to maintain attestation history as HPMS does, we will always have only 1 attestation record per contract. So I guess we just need `findOneByContract`

Reply:
The contract entity has multiple attestations per contract, should that be a 1:1 instead of 1:many?

Reply:
@adaykin Yes.